tools/schema_loader.cc

```diff
@@ -159,7 +160,8 @@ class data_dictionary_impl : public data_dictionary::impl {
         return is_system_keyspace(unwrap(ks).metadata->name());
     }
     virtual const locator::abstract_replication_strategy& get_replication_strategy(data_dictionary::keyspace ks) const override {
-        throw std::bad_function_call();
+        static const locator::local_strategy strategy{locator::replication_strategy_params{locator::replication_strategy_config_options{}, 0}};
+        return strategy;
```

Comment:
I don't understand what this change does. Are you sure it's related to this PR?

Reply:
Yes, it's required, otherwise the code throws (consulted with Botond). 
This previously stubbed function is called on some occasions by the code I introduced (yes, this single `if` statement), hence it can no longer be a stub and needs to return some dummy object.